tachyon/zk/plonk/examples/simple_circuit_test_data.h

```diff
@@ -0,0 +1,1449 @@
+#ifndef TACHYON_ZK_PLONK_EXAMPLES_SIMPLE_CIRCUIT_TEST_DATA_H_
+#define TACHYON_ZK_PLONK_EXAMPLES_SIMPLE_CIRCUIT_TEST_DATA_H_
+
+#include <stdint.h>
+
+#include <utility>
+#include <vector>
+
+#include "tachyon/zk/plonk/examples/circuit_test_data.h"
+#include "tachyon/zk/plonk/layout/floor_planner/simple_floor_planner.h"
+#include "tachyon/zk/plonk/layout/floor_planner/v1/v1_floor_planner.h"
+
+namespace tachyon::zk::plonk::halo2 {
+
+template <typename Circuit, typename PCS, typename LS, typename Enable = void>
+class SimpleTestData : public CircuitTestData<Circuit, PCS, LS> {};
+
+// FloorPlanner = SimpleFloorPlanner
+template <typename Circuit, typename PCS, typename LS>
+class SimpleTestData<
+    Circuit, PCS, LS,
+    std::enable_if_t<std::is_same_v<
+        Circuit, SimpleCircuit<typename PCS::Field, SimpleFloorPlanner>>>>
```

Comment:
It would be nice to change something like below! 

```suggestion
    std::enable_if_t<IsSimpleFloorPlanner<Circuit>>>
```